doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Usage & Billing"
+description: "Provides an overview of various factors that can affect cost."
+menu:
+  main:
+    parent: "administration"
+    weight: 10
+---
+
+From the [console](https://console.materialize.com/) (`Admin` > `Usage &
```

Comment:
I understand this is a first pass, but for a future iteration, it'd be good to stay consistent with existing documentation pages ([Access control](https://materialize.com/docs/manage/access-control/) is again a good example here) and _e.g._ add a short intro.